Source/Tools/FEXLoader/HarnessHelpers.h

```diff
@@ -405,11 +405,7 @@ namespace FEX::HarnessHelper {
         return Result;
       };
 
-#ifndef _WIN32
-      const auto AllocPageSize = FHU::FEX_PAGE_SIZE;
```

Comment:
This would probably be fine to just change to using `getpagesize()` since it is low impact here btw.

Reply:
ack. this is just for me but sure :clown_face: